Rakefile

```diff
@@ -12,6 +12,102 @@
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 6  | regen_individual_profiles       | Regenerates all profiles with all the services. At the end, this task calls                      |
+# |    |                                 | regen_individual_require_files as an enhancement. This task calls clean_individual_profiles      |
+# |    |                                 | as prereq.                                                                                       |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 7  | clean_all_profiles              | Cleans all profiles inside the rollup gem and individual gems.This task has clean_rollup_profiles|
+# |    |                                 | and clean_individual_profiles as prereqs.                                                        |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 8  | clean_rollup_profiles           | Cleans all profiles inside the rollup gem.                                                       |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 9  | clean_individual_profiles       | Cleans all profiles inside individual gems.                                                      |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 10 | regen_all_require_files         | Regenerates the require files for the rollup gem and individual gems. This task has              |
+# |    |                                 | regen_rollup_require_files and regen_individual_require_files as prereqs.                        |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 11 | regen_rollup_require_files      | Regenerates the require files for the rollup gem                                                 |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 12 | regen_individual_require_files  | Regenerates the require files for the individual gems                                            |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 13 | release                         | Release gems. This task has build as a prereq.                                                   |
```

Comment:
we should indicate what controls what gets released when running this task - input file used. And indicate... how would we release an individual gem? btw, based on another issue I was recently looking at, what will be the story for "tagging" the repo when we ship individual gems?

Reply:
Done for the first one. WIP on CI modification